api/v1alpha1/environment_types.go

```diff
@@ -177,9 +177,10 @@ type EnvironmentStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+// +kubebuilder:resource:path=environments,shortName=env
+// +kubebuilder:deprecatedversion:warning="The v1alpha1 version is deprecated and will be automatically migrated to v1beta1"
```

Comment:
Let's change this message to something like:
- `Environment 'v1alpha1' version is deprecated, in favour of 'v1beta1'. 'v1alpha1' will be removed in a future release. See 'https://github.com/redhat-appstudio/application-api/pull/66' for migration steps.`